graylog2-web-interface/src/components/event-definitions/event-definitions/EventDefinitionEntry.tsx

```diff
@@ -82,6 +75,26 @@ const EventDefinitionEntry = ({
   const isScheduled = lodash.get(context, `scheduler.${eventDefinition.id}.is_scheduled`, true);
   const { loadingScopePermissions, scopePermissions } = useGetPermissionsByScope(eventDefinition);
 
+  const isSystemEventDefinition = (): boolean => {
+    return eventDefinition.config.type === 'system-notifications-v1';
+  };
+
+  const titleSuffix = (): JSX.Element | undefined => {
+    if (isSystemEventDefinition()) {
+      return undefined;
+    }
+
+    const plugin = getConditionPlugin(eventDefinition.config.type);
+
+    let suffix = <span>{plugin?.displayName ?? eventDefinition.config.type}</span>;
+
+    if (!isScheduled) {
+      suffix = (<span>{suffix} <Label bsStyle="warning">disabled</Label></span>);
+    }
+
+    return suffix;
```

Comment:
I think it's good to avoid lets if possible. As these two versions of `suffix` don't share anything except for the span tag, we could rewrite it as:

```suggestion
    if(isScheduled) {
      return (<span>{plugin?.displayName ?? eventDefinition.config.type}</span>);
    }
    
    return(<span>{suffix} <Label bsStyle="warning">disabled</Label></span>);
```

Reply:
> I think the general approach looks good, just a few suggestions and questions. However when testing I found an error in the console. For me it happened when opening the System Events Stream and stopping all inputs. When the log came in, the console errored the following: ![Screenshot 2022-12-19 at 17 04 42](https://user-images.githubusercontent.com/3169354/208482970-720bea9c-0f25-467c-a9e5-678096a7b7f8.png)

I checked this issue but I think it's not directly related to this PR.